api/src/wfl/source.clj

```diff
@@ -305,19 +308,20 @@
 ;; but overpolling the TDR increases the chances of:
 ;; - creating many single-row / low-cardinality snapshots
 ;; - locking the dataset / running into dataset locks
-(def ^:private tdr-source-polling-interval-minutes 20)
+(def ^:private tdr-source-default-polling-interval-minutes 20)
 
 (defn ^:private tdr-source-should-poll?
-  "Return true if it's been at least `tdr-source-polling-interval-minutes`
+  "Return true if it's been at least the specified `polling_interval` or, if unspecified, `tdr-source-default-polling-interval-minutes`
```

Comment:
Nit: easier to read if broken up with max 80 chars per line.  Ex.

```
  "Return true if it's been at least the specified `polling_interval`
   or, if unspecified, `tdr-source-default-polling-interval-minutes`
   since `last_checked` -- when we last checked for new rows in the TDR."
```